### ISSUE TITLE
GUI: pressing 'backup' does nothing

### ISSUE THREAD

**Issue by CottonEaster** [teejee2008/aptik#17]
I've selected a folder for backup (that's what 'Backup path' refers to, right - the _destination_ path?) and pressed 'backup' and nothing seems to happen.

Mint 18.3
Aptik GTK 18.5.2  [installed from the Mint repository, so far as I can tell]
Aptil 18.5.2 [installed from the Mint repository, so far as I can tell]

**Comment by teejee2008** [comment]
clicking backup only selects the mode. Now go to the Backup page in sidebar and take the backup.

**Comment by CottonEaster** [comment]
Ah.Thank you. I had seen the Backup page and had assumed that clicking, 'Backup all items' would merely have set all items to be backed-up, i.e. I took the Backup page to comprise merely _options_.

Suggestions: (1) change, 'Backup all items' to, 'Run backup now'; (2) change 'backup path' (on the General page) to something like, 'Destination path for backup'. Also, I think that - on the Backup page - some of the options could use clarification. For, I did not realise that the program was going to backup all the files in /home. Is that what the 'Home - home directory data' option does?
